generate-cpio-rootfs.sh

```diff
@@ -115,7 +115,7 @@ case $1 in
             export ARCH=arm64
         fi
 
-        cp etc/inittab-vexpress etc/inittab
+        cp etc/inittab-hikey etc/inittab
```

Comment:
Yes I think so. Should have been like this before, but `inittab-hikey` and `inittab-vexpress` were the same so it would not make a difference. Now Victor wants login shells on both UART0 (the debug port) and UART3 (the one on the low-speed expansion header), which is a good idea I suppose.